pkg/operator/controllers/guardrails/guardrails_config.go

```diff
@@ -18,9 +18,7 @@ import (
 
 const (
```

Comment:
Can we define all const vars with value starting with "aro.guardrails.*" under operator package and then refer them from it?

Reply:
Hi, asking just to make sure I understand what you're suggesting :) Do you mean doing something like this for the variables that start with `"aro.guardrails"` in this file?

```
package main

import "fmt"

const (
	prefix = "foo"
	suffix = prefix + ".bar"
)

func main() {
	fmt.Println(suffix)
}
```

Or do you mean something else?

Reply:
Not really, for example do you see how _**aro.guardrails.enabled**_ is defined in operator package and then referred here as _**operator.GuardrailsEnabled**_? Something like that if that's possible.
Another example is _**aro.guardrails.deploy.manged**_ which is _**operator.GuardrailsDeployManaged**_.

Reply:
My bad, this is already in the operator package, sorry for the confusion :)